Makefile

```diff
@@ -6,6 +6,10 @@ CURRENT_COMMIT=$(shell git rev-parse --short=7 HEAD)
 OPERATOR_VERSION=$(VERSION_MAJOR).$(VERSION_MINOR).$(COMMIT_NUMBER)-$(CURRENT_COMMIT)
 KUBECTL ?= kubectl
 
+# for boilerplate
+OPERATOR_NAME=route-monitor-operator
```

Comment:
FWIW, boilerplate is set up to [discover](https://github.com/openshift/boilerplate/blob/69bf49e355dd9fe677a6c2d7a295949f6a2998e7/boilerplate/openshift/golang-osd-operator/project.mk#L2) this value from a `config/config.go` file ([example](https://github.com/openshift/aws-account-operator/blob/master/config/config.go)).

(I'll be the first to concede that's pretty janky, especially as I don't recall seeing any project where that config.go is actually used in the operator code. Something to polish later.)